lib/mongoid.rb

```diff
@@ -111,4 +111,14 @@ def client(name)
   #
   # @since 1.0.0
   def_delegators Config, *(Config.public_instance_methods(false) - [ :logger=, :logger ])
+
+  # The default discriminator key for Mongoid
+  #
+  # @example get the base discriminator key
+  #   Mongoid.discriminator_key
+  #
+  # @example set the base discriminator key
+  #   Mongoid.discriminator_key = "test"
+  attr_accessor :discriminator_key
+  self.discriminator_key = '_type'
```

Comment:
Let's change this to a Mongoid config option such that it is settable in the configuration file. Please add the tutorial documentation as well.